src/RealtimeEffectPanel.cpp

```diff
@@ -41,6 +41,7 @@
 #include "effects/RealtimeEffectState.h"
 #include "effects/RealtimeEffectStateUI.h"
 #include "UndoManager.h"
+#include "Prefs.h"
```

Comment:
The last commit in the sequence is still correcting a build error of the previous commit.  I'd like both commits to be compilable.

Reply:
Well, rebase error again. Rebasing takes like half a time on this branch now

Reply:
I merged other approved things to master lately, so please also rebase onto those.